scripts/dev_metrics.py

```
@@ -0,0 +1,452 @@
+# ----------------------------------------------------------------------------
+
+
+def git(*args: str) -> str:
+    return subprocess.run(
+        ["git", *args], capture_output=True, text=True, check=True
+    ).stdout
+
+
+def try_gh(*args: str) -> str | None:
+    try:
+        r = subprocess.run(["gh", *args], capture_output=True, text=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)